import-export-cli/specs/v2/oai3.go

```diff
@@ -100,6 +100,21 @@ func oai3XWso2SandboxEndpoints(exts map[string]interface{}) (*Endpoints, bool, e
 	return nil, true, nil
 }
 
+func oai3XWSO2AuthHeader(exts map[string]interface{}) (string, bool) {
+	if v, ok := exts["x-wso2-auth-header"]; ok {
+        data, ok := v.(json.RawMessage)
```

Comment:
Seems like there is a formatting issue in the method

Reply:
Fixed